client/layout/masterbar/item.tsx

```diff
@@ -52,8 +53,20 @@ class MasterbarItem extends Component< MasterbarItemProps > {
 		url: '',
 	};
 
+	state = {
+		isOpenByTouch: false,
+	};
+
+	componentButtonRef = React.createRef< HTMLButtonElement >();
+	componentDivRef = React.createRef< HTMLDivElement >();
```

Comment:
I had a hard time consolidating these. Typescript always seemed to be angry about trying to do so.